File: GithubGists/app/src/main/java/ru/avystavkin/githubgists/database/AppDatabase.java
package ru.avystavkin.githubgists.database;

import androidx.room.Database;
import androidx.room.RoomDatabase;
import ru.avystavkin.githubgists.models.local.Gist;
import ru.avystavkin.githubgists.models.local.User;
import ru.avystavkin.githubgists.repository.GistDao;

@Database(entities = {Gist.class, User.class}, version = 1)
public abstract class AppDatabase extends RoomDatabase {
    public abstract GistDao gistDao();
}

File: GithubGists/app/src/main/java/ru/avystavkin/githubgists/database/AppDatabase.kt
package ru.avystavkin.githubgists.database

import androidx.room.Database
import androidx.room.RoomDatabase
import ru.avystavkin.githubgists.models.database.Gist
import ru.avystavkin.githubgists.models.database.User
import ru.avystavkin.githubgists.repository.GistDao

@Database(entities = [Gist::class, User::class], version = 1)
abstract class AppDatabase : RoomDatabase() {
    abstract fun gistDao(): GistDao
}
